tests/battDB/test_views.py

```diff
@@ -302,6 +303,90 @@ def test_detail_view(self):
         self.assertContains(response, "<h4> test experiment </h4>")
 
 
+class CreateExperimentTest(TestCase):
+    def setUp(self):
+        self.user = baker.make_recipe(
+            "tests.management.user",
+            username="test_contributor",
+        )
+        self.user.is_active = True
+        self.user.set_password("contribpass")
+        self.user.institution = baker.make_recipe("tests.common.org")
+        self.user.save()
+        group = Group.objects.get(name="Contributor")
+        group.user_set.add(self.user)
+
+    def test_create_update_delete_experiment(self):
+        login_response = self.client.post(
+            "/accounts/login/",
+            {"username": "test_contributor", "password": "contribpass"},
+        )
+        self.assertEqual(login_response.status_code, 302)
+        self.assertEqual(login_response.url, "/")
+
+        form_fields = {
+            "name": "Experiment 1",
+            "date": datetime.now().date(),
+            "exp_type": "constant",
+            "thermal": "none",
+            "summary": "Not long enough",
+        }
+
+        # Invalid Form
+        response = self.client.post(
+            reverse("battDB:New Experiment"),
+            form_fields,
+        )
+        with self.assertRaises(bdb.Experiment.DoesNotExist):
+            bdb.Experiment.objects.get()
+        self.assertEqual(response.status_code, 200)
```

Comment:
Just to check the right error is shown to the user
```suggestion
        self.assertEqual(response.status_code, 200)
        self.assertContains(response, "Ensure this value has at least 20 characters")
```